ParisBNF/abb/BNFabb147.xml

```diff
@@ -42,8 +42,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                              <measure unit="quire">10</measure>
                             <note>The first guard leaf appears to have been designated “A” in the original pagination,
                              here its two pages are assigned numbers “i” and “ii”. Three pieces of paper (<ref target="p2"/>,
-                             <ref target="p3"/>, <ref target="p4"/>) seem to have been accomodated between <locus target="#28">p. 28</locus>
-                           and <locus target="#29">p. 29</locus>. Below they are assigned page numbers 129-130, 131-132, and 133-134, respectively.</note>
+                             <ref target="p3"/>, <ref target="p4"/>) seem to have been accomodated between <locus target="#14v">28</locus>
+                           and <locus target="#15r">29</locus>. Below they are assigned page numbers 129-130 (<locus target="#67"/>),
+                           131-132 (<locus target="#68"/>), and 133-134 (<locus target="#69"/>), respectively.</note>
```

Comment:
If 28 = 14 then
131-132 = 66
133-134 = 67

if you refoliate i-ii as 1-2 then yes, 68 and 69, but then also 28=15v and 29=16r